readthedocs/doc_builder/templates/doc_builder/data.js.tmpl

```diff
@@ -6,5 +6,9 @@ var doc_slug = "{{ slug }}";
 var page_name = "{{ pagename }}";
 var html_theme = "{{ html_theme }}";
 
-READTHEDOCS_DATA["page"] = mkdocs_page_input_path.substr(
-    0, mkdocs_page_input_path.lastIndexOf(READTHEDOCS_DATA.source_suffix));
+if (typeof mkdocs_page_input_path !== 'undefined') {
```

Comment:
Style nit: It looks like all string literals use double quotes in this file, so I'd change it to `"undefined"`.

Reply:
Sounds good.